packages/image-block/src/ImageCaption.tsx

```diff
@@ -0,0 +1,52 @@
+/* (c) Copyright Frontify Ltd., all rights reserved. */
+
+import { RichTextEditor } from '@frontify/fondue';
+import { hasRichTextValue, joinClassNames } from '@frontify/guideline-blocks-shared';
+
+type ImageCaptionProps = {
+    name?: string;
+    isEditing: boolean;
+    description?: string;
+    onNameChange: (value: string) => void;
+    onDescriptionChange: (value: string) => void;
+};
+export const ImageCaption = ({
```

Comment:
Please use guideline design tokens for image title & image caption.

Reply:
added the design tokens. To use the correct style I think its best to do it directly in this ticket: https://app.clickup.com/t/8677a21bv. Wdyt?

Reply:
Good for me 👍🏻